src/image/PhotoCalib.cc

```diff
@@ -85,55 +89,57 @@ double toMagnitudeErr(double instFlux, double instFluxErr, double scale, double
 
 }  // anonymous namespace
 
-// ------------------- Conversions to Maggies -------------------
+// ------------------- Conversions to nanojansky -------------------
 
-double PhotoCalib::instFluxToMaggies(double instFlux, lsst::geom::Point<double, 2> const &point) const {
-    return toMaggies(instFlux, evaluate(point));
+double PhotoCalib::instFluxToNanojansky(double instFlux, lsst::geom::Point<double, 2> const &point) const {
+    return toNanojansky(instFlux, evaluate(point));
 }
 
-double PhotoCalib::instFluxToMaggies(double instFlux) const { return toMaggies(instFlux, _calibrationMean); }
+double PhotoCalib::instFluxToNanojansky(double instFlux) const {
+    return toNanojansky(instFlux, _calibrationMean);
+}
 
-Measurement PhotoCalib::instFluxToMaggies(double instFlux, double instFluxErr,
-                                          lsst::geom::Point<double, 2> const &point) const {
-    double calibration, err, maggies;
+Measurement PhotoCalib::instFluxToNanojansky(double instFlux, double instFluxErr,
+                                             lsst::geom::Point<double, 2> const &point) const {
+    double calibration, err, nanojansky;
     calibration = evaluate(point);
-    maggies = toMaggies(instFlux, calibration);
-    err = toMaggiesErr(instFlux, instFluxErr, calibration, _calibrationErr, maggies);
-    return Measurement(maggies, err);
+    nanojansky = toNanojansky(instFlux, calibration);
+    err = toNanojanskyErr(instFlux, instFluxErr, calibration, _calibrationErr, nanojansky);
+    return Measurement(nanojansky, err);
```

Comment:
This brings up a very good question: what *should* we call the error part of the `value, err[or]` pair in the `Measurement` object. I changed it to `error` as part of this work, because `value, error` seems more sensible to me than `value, err`, but we call the error component `blahErr` everywhere else, so I'm really not sure about it.

I think you're right though, that if we go with `error`, this one should be `error` here.